test/inductor/test_compiled_autograd.py

```diff
@@ -257,6 +257,52 @@ def fn(inputs):
         self.assertNotEqual(grads[1], None)
         self.assertNotEqual(grads[2], None)
 
+    def test_inputs_aliasing_bytecode(self):
```

Comment:
Adding a test to cover the bytecode directly. The behavior of this PR is covered by all tests once we add the inputs list clear on aot side in next PR.